actpool/actQueue.go

```diff
@@ -168,9 +168,9 @@ func (q *actQueue) UpdateQueue(nonce uint64) []*iproto.ActionPb {
 	// Remove all the subsequent actions in the queue starting from that index
 	for ; i < q.index.Len(); i++ {
 		nonce = q.index[i]
-		if transfer := q.items[nonce].GetTransfer(); transfer != nil {
+		if act := q.items[nonce]; act != nil {
```

Comment:
Still need to make sure it is a transfer action instead of a vote. 
if act := q.items[nonce]; act.GetTransfer() != nil {
...